client/blocks/color-scheme-picker/style.scss

```diff
@@ -0,0 +1,16 @@
+.color-scheme-picker .form-radio-with-thumbnail__thumbnail {
+
+	height: 80px;
+	background-image: url( '/calypso/images/color-schemes/theme-default.png' );
+
+	&.is-light-theme {
+
```

Comment:
Nit: I think it'd be more in keeping with the rest of the stylesheets to remove the padded lines here and throughout this file.